components/sidebar/invite_members_button.tsx

```diff
@@ -30,7 +34,7 @@ const InviteMembersButton: React.FC = (): JSX.Element => {
             <ToggleModalButtonRedux
                 accessibilityLabel={intl.formatMessage({id: 'sidebar_left.inviteUsers', defaultMessage: 'Invite Users'})}
                 id='introTextInvite'
-                className='intro-links color--link cursor--pointer'
+                className={`intro-links color--link cursor--pointer${props.className ? ` ${props.className}` : ''}`}
```

Comment:
Not sure if the surrounded backticks around ${props.className} is actually needed?

Reply:
Depends. I didn't want to have the extra space in the case of no passed in class name, and I didn't want to have to use `classnames` or some lib to trim the extra space, so this is the way I thought of to do that. Open to alternate forms of achieving the same thing (including using `classnames` repo).